src/table/pagination-table/components/body/TableTotals.tsx

```diff
@@ -24,7 +24,8 @@ function TableTotals() {
     <TableRow className="sn-table-row">
       {columns.map((column, columnIndex) => {
         const cellCoord: [number, number] = [atTop ? 1 : rows.length + 1, columnIndex];
-        const tabIndex = atTop && columnIndex === 0 && areTabStopsEnabled(keyboard) ? 0 : -1;
+        const tabIndex = atTop && columnIndex === 0 && !keyboard.enabled ? 0 : -1;
```

Comment:
I tried to set tab index to 0 
```
const tabIndex = 0; // atTop && columnIndex === 0 && !keyboard.enabled ? 0 : -1;
```
and I was expecting to focus land on totals cell by default but this happened: 

https://user-images.githubusercontent.com/29652890/226284051-c68aa641-f9cb-4997-b065-709516584d2c.mov

Reply:
I just could not make up to that case (atTop && colIdx and ...) => so i decided to change it to 0 to see what would happen. Just being curious 🤷‍♂️

Reply:
ok I see :)

Reply:
PR looks good, we need to look at this issue in another one